modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/portlet_url.js

```diff
@@ -17,8 +17,6 @@ AUI.add(
 	function(A) {
```

Comment:
There is a bug, util fails when we don't set `basePortletURL`. In that scenario we are hardcoding URL, on line 63, and don't set `http(s)`. `A.URL` handled this, `URL` does not. We need to either check secure and add prefix for hardcoded value.